src/main/webapp/app/entities/exercise/exercise.model.ts

```diff
@@ -61,8 +61,33 @@ export abstract class Exercise implements BaseEntity {
     public numberOfParticipationsWithRatedResult: number;
     public numberOfSuccessfulParticipations: number;
     public averagePoints: number;
+    public numberOfAssessments: number;
 
     protected constructor(type: ExerciseType) {
         this.type = type;
     }
 }
+
+export function getIcon(exerciseType: ExerciseType): string {
+    const icons = {
+        [ExerciseType.PROGRAMMING]: 'keyboard',
+        [ExerciseType.MODELING]: 'project-diagram',
+        [ExerciseType.QUIZ]: 'check-double',
+        [ExerciseType.TEXT]: 'font',
+        [ExerciseType.FILE_UPLOAD]: 'file-upload',
+    };
+
+    return icons[exerciseType];
+}
+
+export function getIconTooltip(exerciseType: ExerciseType): string {
+    const tooltips = {
+        [ExerciseType.PROGRAMMING]: 'This is a programming exercise',
+        [ExerciseType.MODELING]: 'This is a modeling exercise',
+        [ExerciseType.QUIZ]: 'This is a quiz exercise',
+        [ExerciseType.TEXT]: 'This is a text exercise',
+        [ExerciseType.FILE_UPLOAD]: 'This is a file upload exercise',
+    };
```

Comment:
Would it be faster, if we define the tooltips only once in a global constant? and not every time this method gets invoked? (which is probably quite often when you use it from html)